javascript/MLE-Demo/cleanup.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN
+  EXECUTE IMMEDIATE 'DROP PROCEDURE SET_WALLET';
```

Comment:
I do not see such a procedure being created, what part of the scripts here create this procedure?